SpatialGDK/Extras/schema/unreal_metadata.schema

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 package unreal;
 
+import "unreal/gdk/core_types.schema";
+
 component UnrealMetadata {
     id = 100004;
-    option<string> static_path = 1;
+    UnrealObjectRef stably_named_ref = 1; // Exists when entity represents a stably named Actor
```

Comment:
Why not `option<UnrealObjectRef>`?

Reply:
We already have a null state for ObjectRefs and using the `TSchemaOption` everywhere in C++ was proving to be a hassle. Would you want it to be an `option`?

Reply:
Weigh it against the added cost, most actors won't be stably named.

Reply:
I _believe_ that a null object ref and an option would have the same amount of bytes but let me check